### PR TITLE
fixed compatibility for flutter 2.10.x

### PR DESCRIPTION
**Pull request by Harishwarrior**
refer this PR for issue #13

### CODE REVIEW

**Comment by Harishwarrior** [comment]
@munkh-altai @bsneider

File: example/test/widget_test.dart [hunk 1]
```diff
@@ -8,23 +8,23 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
-import 'package:flutter_map_arcgis_example/main.dart';
```
Comment (bsneider) [inline example/test/widget_test.dart:11]: If this file is no longer in use, why not delete it instead of commenting it out?

**Review by Harishwarrior — COMMENTED** [review]
can you review this?

**Review by bsneider — APPROVED** [review]
Looks fine, would do more clean up of files that are not used, but that is not a functional blocker

File: example/test/widget_test.dart (hunk 1 above)
Comment (Harishwarrior) [reply example/test/widget_test.dart:11]: Those are just default test files. Safe to remove. But, I am planning to add test for this project.

File: .fvm/flutter_sdk [hunk 2]
```diff
@@ -0,0 +1 @@
+C:/Users/haris/fvm/versions/stable
```
Comment (pablojimpas) [inline .fvm/flutter_sdk]: Add `.fvm` folder to .gitignore

Comment (Harishwarrior) [reply .fvm/flutter_sdk]: Unfortunately I saw this directory just after i pushed. I'll add it 👍🏻

Comment (Harishwarrior) [reply .fvm/flutter_sdk]: #[7584e0a](https://github.com/khankhulgun/flutter_map_arcgis/pull/14/commits/7584e0a160eeda52d652ac20368de35f55f4c47d)

updated

Comment (pablojimpas) [reply .fvm/flutter_sdk]: Nice, I see that in this commit you've introduced changes in the Gradle wrapper script. Are those intentional? I think it's not meant to be modified.

Comment (Harishwarrior) [reply .fvm/flutter_sdk]: The gitignore used in the project is outdated. Those files you have mentioned shouldn't be tracked in git. Lemme update .gitignore 

https://www.toptal.com/developers/gitignore/api/flutter 

You can refer this for the latest .gitignore.

Comment (Harishwarrior) [reply .fvm/flutter_sdk]: [887fda8](https://github.com/khankhulgun/flutter_map_arcgis/pull/14/commits/887fda8111b35c10713aa5da8eed8c6514fdd4fd)

fixed gitignore

Comment (pablojimpas) [reply .fvm/flutter_sdk]: Perfect! I've just tested your branch in my local android device with flutter 2.10 and it works. I think it's ready to be merged. @munkh-altai

**Comment by munkh-altai** [comment]
Thank you sorry about long time

**Comment by munkh-altai** [comment]
Thank you sorry about long time

**Comment by munkh-altai** [comment]
Thank you sorry about long time

**Pull request merged**